modules/git/repo_commit_test.go

```diff
@@ -50,7 +50,7 @@ func TestGetTagCommitWithSignature(t *testing.T) {
 	assert.NotNil(t, commit)
 	assert.NotNil(t, commit.Signature)
 	// test that signature is not in message
-	assert.Equal(t, "tag", commit.CommitMessage)
+	assert.Equal(t, "signed-commit\n", commit.CommitMessage)
```

Comment:
I'm convinced this `\n` is a bug on its own in the git commit message parsers, but I'm not sure I want to touch that here as well.

Reply:
Pushed a fix for this as well.